src/reaktoro_pse/examples/reaktoro_pse_to_phreeqc_comparison/comparison_utils.py

```diff
@@ -69,23 +70,27 @@ def plot_data_sets(
         ]
     ):
         if key in reaktoro_output_dict[data_type]:
-            reaktor_data_array = np.array(reaktoro_output_dict[data_type][key])
-            phreeqc_data_array = np.array(phreeqc_data[data_type][key])
-
-            percent_dif = (
-                (reaktor_data_array - phreeqc_data_array) / phreeqc_data_array * 100
-            )
-            errors[key] = np.average(np.abs(percent_dif))
-            plot_result(
-                xdata,
-                reaktor_data_array,
-                phreeqc_data_array,
-                xlabel=xlabel,
-                ylabel=key,
-                sweep_name=data_type,
-                save_fig=save_fig,
-            )
-            ax.plot(xdata, percent_dif, color=colors[i], label=key)
+            if key == "pE":
+                phreeqc_key = "pe"
+            else:
+                phreeqc_key = key
+            if phreeqc_key in phreeqc_data[data_type]:
+                reaktor_data_array = np.array(reaktoro_output_dict[data_type][key])
+                phreeqc_data_array = np.array(phreeqc_data[data_type][phreeqc_key])
+                percent_dif = (
+                    (reaktor_data_array - phreeqc_data_array) / phreeqc_data_array * 100
+                )
+                errors[key] = np.average(np.abs(percent_dif))
+                plot_result(
+                    xdata,
+                    reaktor_data_array,
+                    phreeqc_data_array,
+                    xlabel=xlabel,
+                    ylabel=key,
+                    sweep_name=data_type,
+                    save_fig=save_fig,
+                )
+                ax.plot(xdata, percent_dif, color=colors[i], label=key)
```

Comment:
The colors list is shorter than the number of plotted variables. May not cause an error, but just pointing this out.